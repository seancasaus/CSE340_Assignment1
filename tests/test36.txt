Uu P , nIXSu ((h).(((z)|(A)).(L)))|((O)|(((1).(x)).(6))) , xL9X8 ((O).(((N)|(0)).(Z))).((((Z).((K).(R))).(P))|(((n).((k)|(8))).((a).((r).(Y))))) , OHdnP (W)|(((a).(A)).(((q).(8)).(((W)|(H))|(i)))) , z3b8X (((0)|((T).(y)))|(((q)|(s))|(((R).(P))|(f))))|((((w)|((A)|(F)))|((x).(Q))).((K)|(((E).(5)).(5)))) , FkzMa ((((g).((d)|(A))).(A))|(W))|(q) , UO u , aX V , I0S6p P , gC4qb (((r).(N)).(((o).(H)).(X)))|((L).((((o)|(w))|((_)|(w)))|(w))) #
"LwWq0hALPVhzLVVxQE55VWLVPO0ZZKRPWOLhzLFE55gAArNoHXPONZZKRPuONZnkarYPuPhzLPOLwrNoHXqPhALPLuPaAq8HVPPgAAuquVVxQE551x6uONZZKRPLLwuWugAArNoHXgAAPxQKVwE55PaAq8WuVxQKO0Zn8arYwK " 