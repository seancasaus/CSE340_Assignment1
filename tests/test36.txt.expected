gC4qb , "Lw"
OHdnP , "W"
z3b8X , "q"
z3b8X , "0"
nIXSu , "hAL"
Uu , "P"
aX , "V"
nIXSu , "hzL"
aX , "V"
aX , "V"
z3b8X , "xQE55"
aX , "V"
OHdnP , "W"
gC4qb , "L"
aX , "V"
Uu , "P"
xL9X8 , "O0ZZKRP"
OHdnP , "W"
nIXSu , "O"
gC4qb , "L"
nIXSu , "hzL"
z3b8X , "FE55"
FkzMa , "gAA"
gC4qb , "rNoHX"
Uu , "P"
xL9X8 , "ONZZKRP"
UO , "u"
xL9X8 , "ONZnkarY"
Uu , "P"
UO , "u"
Uu , "P"
nIXSu , "hzL"
Uu , "P"
nIXSu , "O"
gC4qb , "Lw"
gC4qb , "rNoHX"
z3b8X , "q"
Uu , "P"
nIXSu , "hAL"
Uu , "P"
gC4qb , "L"
UO , "u"
Uu , "P"
OHdnP , "aAq8H"
aX , "V"
Uu , "P"
Uu , "P"
FkzMa , "gAA"
UO , "u"
z3b8X , "q"
UO , "u"
aX , "V"
aX , "V"
z3b8X , "xQE55"
nIXSu , "1x6"
UO , "u"
xL9X8 , "ONZZKRP"
gC4qb , "L"
gC4qb , "Lw"
UO , "u"
OHdnP , "W"
UO , "u"
FkzMa , "gAA"
gC4qb , "rNoHX"
FkzMa , "gAA"
Uu , "P"
z3b8X , "xQK"
aX , "V"
z3b8X , "wE55"
Uu , "P"
OHdnP , "aAq8W"
UO , "u"
aX , "V"
z3b8X , "xQK"
xL9X8 , "O0Zn8arY"
z3b8X , "wK"
